code/modules/hydroponics/grown/replicapod.dm

```diff
@@ -119,6 +121,7 @@
 		podman.hardset_dna(null,null,podman.real_name,blood_type, new /datum/species/pod,features)//Discard SE's and UI's, podman cloning is inaccurate, and always make them a podman
 		podman.set_cloned_appearance()
 		log_cloning("[key_name(mind)] cloned as a podman via [src] in [parent] at [AREACOORD(parent)].")
+		podman.notify_ghost_cloning("You have been cloned by a replica pod and can now enter your new body.", source = podman)
```

Comment:
imo, the message should be sent when the seed is PLANTED, not when it's HARVESTED

we're already immediately putting the ghost in their new podperson body upon the harvesting of the seed anyway, right? why do we need to send them a message about it?